### PR TITLE
fix: Add Jose Jimmy to the list

### LINKED ISSUE #842: add Achilles8055
Screenshot (882)

### PR DESCRIPTION
**Pull request by Achilles8055**
resolve #842

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -1582,5 +1582,11 @@
     "githubUsername": "ChoudharyTara",
     "imageUrl": "https://user-images.githubusercontent.com/87922322/141666358-ffc20544-4424-4e6e-a702-9e0912082b44.png",
     "issueNumber": 835
-}
+ },
```
Comment (Panquesito7) [inline data.json:1585]:
```suggestion
},
```

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please fix what @Panquesito7 suggested

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍
By the way, I liked your profile 🎉 it's awesome.

**Comment by Achilles8055** [comment]
Thanks,  David Leal for the compliment. It means a lot.  Looking forward to
learn and contribute more  😊😊

**Pull request merged**